.CI/create_feedstocks.py

```diff
@@ -254,10 +252,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
```

Comment:
Turns out Travis CI's `git` is too old to support this. Our `git` is new enough to support this. So am proposing we switch to it as part of this change with PR ( https://github.com/conda-forge/staged-recipes/pull/1464 ).